src/app/context/AuthContext.tsx

```diff
@@ -27,9 +31,16 @@ interface AuthContextType {
 
 const AuthContext = createContext({} as AuthContextType);
 
+// @TODO: open new issue
```

Comment:
@dylancom I just saw this...  
https://github.com/getAlby/lightning-browser-extension/blob/chore/yarn-action-3-only/src/app/context/AccountsContext.tsx  
Should we move everything to that?

Reply:
Yes, we could combine those.

Reply:
Added a todo